### PR TITLE
Add fly tool to advanced tools panel

### PR DESCRIPTION
**Pull request by lachlansleight**
This is basically a nicer version of the fly tool that I described in my tutorial.

It features the following changes:

  * Added a new tool to the SketchControls/SketchSurface object in the Main Scene
  * Added a new panel button to the AdvancedToolsPanel and AdvancedToolsPanel_Mobile prefabs
  * Added the necessary enum values to BaseTool.cs and InputManager.cs
  * Added the necessary input handling to ControllerInfo.cs
  * Added a new icon for the Fly Tool

## Note that this PR includes a modification to the Main scene! This could be avoided in future by making SketchSurface a prefab

### CODE REVIEW

**Comment by mikeage** [comment]
Hi @lachlansleight , I hope you don't mind but I merged `main` into your branch, to get the CI builds to run.

File: Assets/Scripts/Tools/FlyTool.cs [hunk 1]
```diff
@@ -0,0 +1,149 @@
+﻿using System.Collections;
```
Comment (mikeage) [inline Assets/Scripts/Tools/FlyTool.cs:1]: Can you please add a header, similar to that of the other files, with `Copyright (c) 2021 The Open Brush Authors` in place of the Google/TiltBrush copyright? 

After that, we're ready to merge this

Comment (lachlansleight) [reply Assets/Scripts/Tools/FlyTool.cs]: Done - I literally just copied the existing license comment, which I'm assuming is right?

**Comment by mikeage** [comment]
Looks great, thanks!

**Pull request merged**